content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -95,8 +95,10 @@ to learn more.
 ### Background cascading deletion {#background-deletion}
 
 In background cascading deletion, the Kubernetes API server deletes the owner
-object immediately and the controller cleans up the dependent objects in
-the background. By default, Kubernetes uses background cascading deletion unless
+object immediately and the controller (like the custom or default controller)
+cleans up the dependent objects in the background.
```

Comment:
Perhaps it's better to remove "(like the custom or default controller)" to avoid confusion
```suggestion
object immediately and the controller cleans up the dependent objects in the background.
```

Reply:
I included it due to a complaint in the referenced issue. The issue creator was using a custom controller and wasn’t sure if the custom one was also supported. Kindly see this [review comment](https://github.com/kubernetes/website/pull/47977/files#r1766248775) where I specified the name of the controller.